src/options.go

```diff
@@ -1726,7 +1726,7 @@ func parsePreviewWindowImpl(opts *previewOpts, input string) error {
 	var err error
 	tokenRegex := regexp.MustCompile(`[:,]*(<([1-9][0-9]*)\(([^)<]+)\)|[^,:]+)`)
 	sizeRegex := regexp.MustCompile("^[0-9]+%?$")
-	offsetRegex := regexp.MustCompile(`^(\+{-?[0-9]+})?([+-][0-9]+)*(-?/[1-9][0-9]*)?$`)
+	offsetRegex := regexp.MustCompile(`^(\+{-?[0-9n]+})?([+-][0-9]+)*(-?/[1-9][0-9]*)?$`)
```

Comment:
We need to refine the expression not to allow things like `{-nnn123nnn}`.

```suggestion
	offsetRegex := regexp.MustCompile(`^(\+{(-?[0-9]+|n)})?([+-][0-9]+)*(-?/[1-9][0-9]*)?$`)
```

Reply:
Thanks for the suggestion! I was thinking of the same thing but didn't want to risk introducing new matching groups in case it breaks the code that depends on this regex somehow.

Let me give your suggestion a try and re-upload, but yes agreed it looks much nicer and more focused.

Reply:
Tried it and all tests are passing! Rejects all other illegal values like `{-n}`, `{12n}`, or `{-54n1}`!

Reply:
Updated and re-uploaded the PR! Didn't want to click 'Commit suggestion' since it would've created another commit on top of the existing 2 and polluted the git tree.

Please let me know if it's all good to merge!